src/main/deltachat.js

```diff
@@ -145,6 +145,17 @@ class DeltaChatController {
     if (typeof this._render === 'function') this._render()
   }
 
+  /**
+   * Dispatched when showing the AboutDialog (owned by ScreenController)
+   */
+  getInfo () {
+    if (this.ready === true) {
+      return this._dc.getInfo()
+    } else {
+      return DeltaChat.getSystemInfo()
```

Comment:
This returns basic system information, which is a stripped down version of `dc.getInfo()`

Reply:
@Jikstra We could skip the top line and just do `return DeltaChat.getSystemInfo()`.